content/en/getting_started/containers/datadog_operator.md

```diff
@@ -62,6 +62,39 @@ The [Datadog Operator][1] is an open source [Kubernetes Operator][2] that enable
   kubectl apply -f /path/to/your/datadog-agent.yaml
   ```
 
+### Running Agents in a Single Container
+
+<div class="alert alert-warning">Available in Operator 1.4.0 or later</div>
+
+By default, Operator creates an agent daemonset with pods running multiple agent containers. Operator 1.4.0 introduces a configuration which allows users to run agents in a single container. This feature is only applicable when privileged agents are not required.
+
+To enabled this feature add `global.containerStrategy: single` configuration to the `DatadogAgent` manifest:
+
+  ```yaml
```

Comment:
```suggestion
{{< highlight yaml "hl_lines=7" >}}
```

Reply:
Suggesting to replace the code fences -- using the `highlight` setting here allows the line to be highlighted for the reader. Screenshot:

<img width="840" alt="containerStrategy_setting" src="https://github.com/DataDog/documentation/assets/36318703/ff26831a-536a-4470-b329-209f2f184361">